topi/python/topi/x86/conv2d.py

```diff
@@ -327,11 +327,16 @@ def _alter_conv2d_layout(attrs, inputs, tinfo, F):
 
     copy_inputs = [s for s in inputs]
     new_attrs = {k : attrs[k] for k in attrs.keys()}
+
+    if F == tvm.relay.op:
```

Comment:
@merrymercy @cbalint13 This call fails in python3. You need to `import relay from tvm` first
```
$ python3
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tvm
>>> tvm.relay.op
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'tvm' has no attribute 'relay'
```

Reply:
@apivovarov ,
  * Could make it in a new PR ?

Reply:
If we had nnvm compilation tests then your PR would not pass CI tests and you needed to fix your code.

Reply:
@apivovarov ,

Agree.

* Will add as a new PR today, with testcase using nnvm symbols instead of relay.op
* I'll keep you on Cc

Reply:
@cbalint13 I'm not sure we want to import `relay` if user decided to use `nnvm.compiler.build`.
Can we use string comparison here? e.g.
```
if F.__name__ == 'tvm.relay.op':
```

Reply:
continue discussion in
https://github.com/dmlc/tvm/pull/3044